src/context.ts

```diff
@@ -147,6 +147,17 @@ export class PageContext {
       return await resolveReactRoutes(this)
     if (this.options.resolver === 'solid')
       return await resolveSolidRoutes(this)
+    if (isPageResolverFunc(this.options.resolver))
+      return await this.options.resolver(this)
+    if (isPageResolverImport(this.options.resolver)) {
+      try {
+        const customResolver: PageResolverFunc = await import(this.options.resolver)
+        return await customResolver(this)
+      } catch (e) {
+        throw new Error(`Unable to resolve router plugin ${this.options.resolver
+        }`)
+      }
+    }
```

Comment:
I think we don't need this, user can import and directly use the resolver in the config file.

Reply:
My thinking is someone creates a 3rd party resolver as a NPM package. People might just want to use the string. Happy to remove if not required

Reply:
`await import` might have some problem with CJS, so I think is better to remove it.